### DISCUSSION TITLE
secrets not working after repo ownership transfer

### DISCUSSION THREAD

**Discussion by chenb67** [community/community#25040] [Actions]
<p>secrets that were set before the transfer are working a new ones are not.</p>
<p>I have full access in all the repos</p>

**Comment by weide-zhou** [comment]
<p>Hi <a class="mention" href="/chenb67">@chenb67</a> ,</p>
<p>It’s not repro on my side, after repo transfer, i create a new secrets and it works well(i used PAT for the value).</p>
<p>How did you create the secrets after transfter? Please check the secrets(same value) in another workflow firstly and make sure it works.</p>
<p>Thanks.</p>

**Comment by chenb67** [comment]
<p>Hi,</p>
<p>Thanks for the help</p>
<p>Tried it with a new workflow the same issue - secrets that were created before the transfer are working ok and the new ones are not.</p>

**Comment by chenb67** [comment, marked as answer]
<p>I removed the secrets and readded them and it solved it</p>

**Comment by jr200** [comment]
confirming this is still broken. 6 years later, need to remove and re add the secret still
